src/components/user/SignInForm/index.js

```diff
@@ -1,46 +1,75 @@
 
-const SignInForm = ({ handleSubmit, signUpLink }) => (
-  <View style={styles.container} keyboardShouldPersistTaps={'handled'} onSubmit={handleSubmit}>
-    <View style={styles.formContainer}>
-    <View style={styles.field}>
-      <Text style={styles.label}>EMAIL</Text>
-      <Field
-        name='email'
-        component={Input}
-      />
-    </View>
-    <View style={styles.field}>
-      <Text style={styles.label}>PASSWORD</Text>
-      <Field
-        name='password'
-        component={Input}
-        password
-      />
-    </View>    
-    </View>    
-    <View style={styles.buttonsContainer}>
-      <ActionButton title='SIGN IN' backgroundColor={black} textColor={white} onPress={handleSubmit}/>
-      <TouchableOpacity>  
-        <Text style={styles.forgotPasswordText}>Forgot your password?</Text>
-      </TouchableOpacity>
-      <TouchableOpacity>
-        <Text style={styles.connectWithFacebookText}>CONNECT WITH FACEBOOK</Text>
-      </TouchableOpacity>
-    </View>
-    <View style={styles.dividerContainer}/>
-      <TouchableOpacity onPress={signUpLink}>
-        <Text style={styles.buttonText}>SIGN UP</Text>
-      </TouchableOpacity>
-  </View>
-);
+class SignInForm extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleFacebookSignIn(error, result, facebookSignIn){
+    if (error) {
+      alertErrors('login has error: ' + result.error);
+    } else if (result.isCancelled) {
+      alertErrors('login is cancelled.');
+    } else {
+      AccessToken.getCurrentAccessToken().then(
+        (data) => {
+          facebookSignIn(data.accessToken.toString());
+        }
+      )
+    }
+  }
+
+  render() {
+    const { handleSubmit, goToSignUp, facebookSignIn } = this.props;
+
+    return(
+      <View style={styles.container} keyboardShouldPersistTaps={'handled'} onSubmit={handleSubmit}>
+        <View style={styles.formContainer}>
+          <View style={styles.field}>
+            <Text style={styles.label}>EMAIL</Text>
+            <Field
+              name='email'
+              component={Input}
+            />
+          </View>
+          <View style={styles.field}>
+            <Text style={styles.label}>PASSWORD</Text>
+            <Field
+              name='password'
+              component={Input}
+              password
+            />
+          </View>    
+        </View>    
+        <View style={styles.buttonsContainer}>
+          <ActionButton title='SIGN IN' backgroundColor={black} textColor={white} onPress={handleSubmit}/>
+          <TouchableOpacity>  
+            <Text style={styles.forgotPasswordText}>Forgot your password?</Text>
+          </TouchableOpacity>
+          <View style={styles.facebookSignInContainer}>
+            <LoginButton
+              readPermissions={['email']}
+              onLoginFinished={(error, result) => this.handleFacebookSignIn(error,result,facebookSignIn)}
+              onLogoutFinished={() => console.log('logout.')}/>
```

Comment:
Would prefer to have
```suggestion
              onLogoutFinished={() => {}}/>
```